### DISCUSSION TITLE
Auxiliary function in output transform

### DISCUSSION THREAD

**Discussion by ghost** [lululxvi/deepxde#1722] [Q&A]
Hello,

I am using the auxiliary_var_function argument in DeepXDE.data.PDE to enable interpolation and indexing with numpy functions instead of Tensorflow in the hope of it reducing the training time. I have an output transform which requires the same auxiliary_var_function, however it seems like it does not work the same way as the PDE function (i.e. add auxiliary_vars as an argument to pde: pde(inputs, outputs, auxiliary_vars) ). Is there a specific way to use the same auxiliary function that receives inputs from the model and used in the PDE in the output transform?

**Comment by lululxvi** [comment]
It is not supported at this moment. Need to modify the source code.
